contributing/checklist.md

```diff
@@ -174,6 +174,21 @@ See Apple's [Accessibility Programming Guide for iOS](https://developer.apple.co
 1. Test your control on a device in VoiceOver mode and ensure the bahavior is at least as robust as UIKit.
 1. Enter YES, NO or N/A
 
+### Dynamic Type Support (If possible)
+
+
+Any component that has text, interacts with text, or lays itself out in relation to text must implement Dynamic Type.
```

Comment:
s/must implement/should implement/
(We may decide some components should not for design / feasibility reasons)

Reply:
Done